boards/arm/stm32f746g_disco/pinmux.c

```diff
@@ -14,14 +14,6 @@
 
 /* pin assignments for STM32F746G-DISCO board */
 static const struct pin_config pinconf[] = {
-#if DT_NODE_HAS_STATUS(DT_NODELABEL(usart1), okay) && CONFIG_SERIAL
```

Comment:
You're mixing 2 boards here

Reply:
I'm afraid you're going to have to elaborate a bit more, I don't understand where the mixing happens ?

Reply:
Done in latest push